content/docs/for-developers/sending-email/using-handlebars.md

```diff
@@ -11,33 +11,41 @@ seo:
   description:
 ---
 
-## Handlebars overview
+Twilio SendGrid [Dynamic Transactional Templates]({{root_url}}/ui/sending-email/how-to-send-an-email-with-dynamic-transactional-templates/) and [Marketing Campaigns designs]({{root_url}}/ui/sending-email/working-with-marketing-campaigns-email-designs/) support the [Handlebars](https://handlebarsjs.com/) templating language to render dynamic data.
 
-[Handlebars.js syntax](https://handlebarsjs.com/) provides a simple, powerful way to include dynamic content directly in email templates. Handlebars.js syntax allows all of this dynamic templating to occur outside of code, meaning changes are done quickly in the template, with no update to a code base required.
+Dynamic data allows you to personalize the messages in your templates by inserting customers' names, addresses, and other data to make an email relevant to each individual recipient. For example, if you have a customer's name stored in a JSON property called `"name"`, you can insert the property's value into a template by adding `{{ name }}` wherever you want the customer's name to appear.
 
-This page uses examples from the [dynamic-template section of our email templates GitHub repo](https://github.com/sendgrid/email-templates/tree/master/dynamic-templates).
+Handlebars syntax allows all of this dynamic templating to occur outside of your code base, meaning changes are done quickly in the template with no update to a code base required.
 
 <call-out>
 
-For our API reference, see [Mail Send with Dynamic Transactional Templates](https://sendgrid.api-docs.io/v3.0/transactional-templates).
+If you prefer to use your own templating system, you can still insert dynamic data using [Substitution Tags](/ui/sending-email/substitution-and-section-tags/).
```

Comment:
May be more clear to qualify substitution tags e.g. "using Substitution Tags, which are stored with your contacts." or something like that